### PR TITLE
Publish oc-mirror for all arches for ocp >= 4.14

### PR DESCRIPTION
**Pull request by locriandev**
Ref. https://issues.redhat.com/browse/ART-6820 and https://issues.redhat.com/browse/ART-6863

### CODE REVIEW

File: pyartcd/pyartcd/pipelines/promote.py [hunk 1]
```diff
@@ -424,7 +424,9 @@ async def publish_client(self, working_dir, from_release_tag, release_name, buil
             else:
                 self._logger.error(f"Error get {tarball} image from release pullspec")
 
-        if build_arch == 'x86_64':
+        # Starting from 4.14, oc-mirror will be synced for all arches. See ART-6820 and ART-6863
+        major, minor = util.isolate_major_minor_in_group(self.group)
+        if major > 4 or minor >= 14 or build_arch == 'x86_64':
```
Comment (joepvd) [inline pyartcd/pyartcd/pipelines/promote.py:429]: I think this also triggers for 4.6, as major > 4

**Review by joepvd — APPROVED** [review]
/lgtm

File: pyartcd/pyartcd/pipelines/promote.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (thegreyd) [reply pyartcd/pyartcd/pipelines/promote.py:429]: A stylistic suggestion - being specific helps in understanding the exact branching condition we're aiming for, anybody who reads this piece of code won't need to know 3 ends at 3.11 or 5.X isn't available etc.

[the remark this answers is not in the document]
Comment (locriandev) [reply pyartcd/pyartcd/pipelines/promote.py:429]: I explicitly wanted to include 5.X as well by saying "anything that came after 4.14 will have this change".

[the remark this answers is not in the document]
Comment (thegreyd) [reply pyartcd/pyartcd/pipelines/promote.py:429]: Right. Don't have a strong preference, it can be `major >= 4`

**Pull request merged**